.travis.yml

```diff
@@ -1,39 +0,0 @@
-jdk:
-- oraclejdk11
-dist: trusty
-language: scala
-scala:
-- 2.13.5
-sudo: required
-services:
-- docker
-before_install:
-- sh docker/run-opendj.sh start
-- sh docker/run-postgres.sh start 15432
-- >-
-  git clone https://github.com/awslabs/git-secrets.git git-secrets &&
-  pushd git-secrets > /dev/null &&
-  git checkout ad82d68ee924906a0401dfd48de5057731a9bc84 &&
-  export PATH="${PATH}:${PWD}" &&
-  popd > /dev/null
-script:
-- >-
-  ./minnie-kenny.sh --force &&
-  git secrets --scan-history
-- export SBT_OPTS="-J-Xmx3g -Denv.type=test -Ddirectory.url=ldap://localhost:3389
-  -Ddirectory.password=testtesttest -Dpostgres.host=localhost -Dpostgres.port=15432"
-- sbt clean "testOnly -- -n org.broadinstitute.tags.SchemaInit"
-- docker restart opendj
-- sleep 40
-- sbt coverage "testOnly -- -l org.broadinstitute.tags.SchemaInit" coverageReport
-- bash scripts/gen_java_client.sh
-after_success:
-- export SBT_OPTS="-J-Xmx3g"
-- sbt coveralls
```

Comment:
i _think_ we can remove this dependency https://github.com/broadinstitute/sam/blob/c34ff18ddb109aa15e4f4cade8377f582414c86c/project/plugins.sbt#L5